docs/website/mmp-errors.md

```diff
@@ -174,7 +174,7 @@ An unexpected error occured when processing an assembly.
 
 The assembly causing the issue is named in the error message. In order to fix this issue the assembly will need to be provided in a [bug report](https://bugzilla.xamarin.com) along with a complete build log with verbosity enabled (i.e. `-v -v -v -v` in the **Additional mtouch arguments**).
 
-### <a name="MM2104"/>MM2104: Unable to link assembly '{0}' as it is mixed-mode.
+### <a name="MM2104"/>MM2104: Unable to link assembly '{0}' as it is mixed-mode. See https://msdn.microsoft.com/en-us/library/x0w2664k.aspx for more information on mixed-mode assemblies.
```

Comment:
It meant more _description_, not part of the error message - which should be identical to the tool's output

and it would be nicer with a clickable (in markdown) link :)